app/models/account/states.rb

```diff
@@ -94,6 +94,7 @@ module Account::States
       end
     end
 
+    scope :without_suspended, -> { where.not(state: :suspended) }
```

Comment:
`unsuspended`?

Reply:
@hallelujah or `not_suspended`, but I wrote it like this to make it consistent with the other scope (the one for `scheduled_for_deletion`) that we already have in `master`:
https://github.com/3scale/porta/blob/8474333f885e82638aecbb2b047302e7f19444ea/app/models/account/states.rb#L97

Reply:
Which is literally the next line after this new one that I wrote here hehe 😄

Reply:
Ah I got it, well my english is kinda bad to know those differences. :/